.github/workflows/deps-updates.yml

```
@@ -7,6 +7,7 @@ on:
 jobs:
   deps-updates:
     runs-on: ubuntu-latest
+    if: ${{ secrets.RENOVATE_TOKEN != '' }}
```

actionlint (1.7.4)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)